README.md

```
@@ -0,0 +1,422 @@
+    - [x] 시작 시간에 유효하지 않은 값이 입력될 경우, 예외가 발생한다.
+- [x] 예약 가능한 시간을 모두 조회할 수 있다.
+- [x] 관리자는 예약 가능한 시간을 삭제할 수 있다.
+    - [x] 삭제하려는 시간에 대한 예약이 존재할 경우, 예외가 발생한다.
+
+### 테마
+
+- [x] 관리자는 테마를 추가할 수 있다.
+- [x] 테마를 모두 조회할 수 있다.
+- [x] 관리자는 테마를 삭제할 수 있다.
+- [ ] 인기 테마를 조회할 수 있다
+    - [ ] 인기 테마 : 최근 일주일을 기준으로 해당 기간내에 예약이 많은 테마 10개
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)